Sage/carbon
# docs(toast): fix logic with stacked-delayed example

fixes #3087

### Proposed behaviour
![toast_story_fix](https://user-images.githubusercontent.com/56251247/96707791-3deb9600-1390-11eb-82a6-d1c418bb8d1a.gif)

### Current behaviour

![Screen Recording 2020-07-22 at 14 55 34](https://user-images.githubusercontent.com/53473905/88185721-2fc09900-cc2c-11ea-8ae1-a858066a41c1.gif)

### Checklist



### Additional context
N/A

### Testing instructions
- `git checkout FE-2997-toast_component_error`
- `npm start` to start Storybook
- Navigate to Design System -> Toast - > stacked-delayed
- Example should now work as expected
